kitsune/users/tests/test_templates.py

```diff
@@ -153,6 +153,20 @@ def test_bio_links_no_img(self):
         self.assertEqual(200, r.status_code)
         assert b"<p>my dude image </p>" in r.content
 
+    def test_bio_links_abbr(self):
+        self.profile.bio = '<abbr title="my full thing">MFT></abbr>'
+        self.profile.save()
+        r = self.client.get(reverse("users.profile", args=[self.u.username]))
+        self.assertEqual(200, r.status_code)
+        assert b'<abbr title="my full thing">MFT></abbr>' in r.content
+
+    def test_bio_links_acronym(self):
```

Comment:
I don't think that the `links` part of the name fits here.
```suggestion
    def test_bio_acronym(self):
```